chromium/utils/translate-en-messages.py

```diff
@@ -1,139 +1,119 @@
-'''
+"""
 Script:       translate-en-messages.py
 Version:      2024.5.14.1
-Description:  Translate msg's from en/messages.json to [[output_langs]/messages.json]
+Description:  Translate messages from en/messages.json to other language directories.
 Author:       Adam Lui
+Review:       Hexakleo
 Homepage:     https://github.com/adamlui/python-utils
-'''
+"""
 
-import os, json
-from sys import stdout # for dynamic prints
+import os
+import json
+from sys import stdout
```

Comment:
For this it is condensed to save lines

Reply:
Go to https://github.com/adamlui/python-utils, there you can create multiple PRs for even more merged ones to collect, each one make it related to one change